batch7crud-roll1/src/main/java/com/lftechnology/batch7crud/validator/EmployeeValidator.java

```diff
@@ -0,0 +1,57 @@
+package com.lftechnology.batch7crud.validator;
+
+import com.lftechnology.batch7crud.entity.Employee;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.lftechnology.batch7crud.constant.EntityConstant.*;
+import static com.lftechnology.batch7crud.constant.EntityConstant.ADDRESS;
+import static com.lftechnology.batch7crud.constant.EntityConstant.AGE;
+
+/**
+ * EmployeeValidator validates Employee related user input data
+ * It checks if input data are empty or null
+ *
+ * @Author Kiran Pariyar <kiranpariyar@lftechnology.com>
+ * Created on 1/25/16.
+ */
+
+public class EmployeeValidator implements Validator<Employee> {
+
+  @Override
+  public Map<String, String> validate(Employee employee) {
+
+    Map<String, String> errors = new HashMap<>();
+
+    if (isNullOrEmpty(employee.getUserName().trim())) {
+      errors.put(USER_NAME, "username cannot be empty");
+    }
+
+    if (isNullOrEmpty(employee.getPassword().trim())) {
+      errors.put(PASSWORD, "password cannot be empty");
+    }
+
+    if (isNullOrEmpty(employee.getFullName().trim())) {
```

Comment:
instead of trim in all the cases what if you provide the trim before checking isEmpty